packages/redux-dynostore-core/src/escapeIdentifier.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2018, IOOF Holdings Limited.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import isPlainObject from 'lodash.isplainobject'
+
+const ESCAPE_TABLE = {
+  '.': '%%DOT_TOKEN%%',
```

Comment:
Part of me thinks this is overly complex and reduces readability in debugging (question, do you see this in the redux dev tools?)

can we just escape with a preceding `\`, then when splitting on the name, use a regex like `(?<!\\)\.` ?